sdk/core/azure-core-amqp/vcpkg/vcpkg.json

```diff
@@ -20,7 +20,7 @@
     {
       "name": "azure-core-cpp",
       "default-features": false,
-      "version>=": "1.10.0"
```

Comment:
Please extract this change and make a separate PR, we'll merge it into main and I'll start Core release with that SHA.